Automattic/jetpack
# Autoloader: use all of the map files to populate the global maps

#### Changes proposed in this Pull Request:
* The latest autoloader needs to examine all of the classmap and filemap files created by the new autoloaders and populate the globals. In the `enqueue_files()` function, loop through all of the active plugins and examine the classmap and filemap files.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* This changes an existing part of Jetpack.

#### Testing instructions:
* Try using multiple plugins that require the autoloader, and verify that no errors occur.

#### Proposed changelog entry for your changes:
* N/A
